Packs/AnythingLLM/Integrations/AnythingLLM/AnythingLLM.py

```diff
@@ -214,25 +209,34 @@ def workspace_new(self, workspace: str):
             demisto.debug(msg)
             raise Exception(msg)
 
-
     def workspace_chat(self, workspace: str, message: str, mode: str):
         return self._chat(workspace, message, mode, "chat")
 
-
     def workspace_stream_chat(self, workspace: str, message: str, mode: str):
         return self._chat(workspace, message, mode, "stream-chat")
 
-
     def workspace_list(self):
         return self._list("workspaces")
 
+    def thread_list(self, workspace: str):
+        try:
+            slug = workspace_slug(workspace, self.workspace_list())
+            response = self._http_request(
+                method="GET",
+                url_suffix=f"/v1/workspace/{slug}",
+            )
+        except Exception as e:
+            msg = f"AnythingLLM: thread_list: exception listing workspace threads - {str(e)}"
```

Comment:
```suggestion
            msg = f"AnythingLLM: thread_list: exception listing workspace threads - {e}"
```
The f-string should handle converting the value to a string.

Reply:
Made the change globally in the integration and all automations